2017-02-10 12:35:04,518 - [INFO] - from play in pool-5-thread-1 
Listening for HTTP on /0:0:0:0:0:0:0:0:9000

2017-02-10 12:35:49 -0700 [INFO] from play in play-internal-execution-context-1 - 
datasource [jdbc:mysql://127.0.0.1/advWeb] bound to JNDI as DefaultDS
2017-02-10 12:35:49 -0700 [INFO] from play in play-internal-execution-context-1 - 
database [default] connected at jdbc:mysql://127.0.0.1/advWeb
2017-02-10 12:35:49 -0700 [WARN] from org.hibernate.ejb.HibernatePersistence in play-internal-execution-context-1 - 
HHH015016: Encountered a deprecated javax.persistence.spi.PersistenceProvider [org.hibernate.ejb.HibernatePersistence]; use [org.hibernate.jpa.HibernatePersistenceProvider] instead.
2017-02-10 12:35:49 -0700 [WARN] from org.hibernate.ejb.HibernatePersistence in play-internal-execution-context-1 - 
HHH015016: Encountered a deprecated javax.persistence.spi.PersistenceProvider [org.hibernate.ejb.HibernatePersistence]; use [org.hibernate.jpa.HibernatePersistenceProvider] instead.
2017-02-10 12:35:49 -0700 [WARN] from org.hibernate.ejb.HibernatePersistence in play-internal-execution-context-1 - 
HHH015016: Encountered a deprecated javax.persistence.spi.PersistenceProvider [org.hibernate.ejb.HibernatePersistence]; use [org.hibernate.jpa.HibernatePersistenceProvider] instead.
2017-02-10 12:35:50 -0700 [INFO] from org.hibernate.jpa.internal.util.LogHelper in play-internal-execution-context-1 - 
HHH000204: Processing PersistenceUnitInfo [
	name: defaultPersistenceUnit
	...]
2017-02-10 12:35:50 -0700 [INFO] from org.hibernate.Version in play-internal-execution-context-1 - 
HHH000412: Hibernate Core {4.3.10.Final}
2017-02-10 12:35:50 -0700 [INFO] from org.hibernate.cfg.Environment in play-internal-execution-context-1 - 
HHH000206: hibernate.properties not found
2017-02-10 12:35:50 -0700 [INFO] from org.hibernate.cfg.Environment in play-internal-execution-context-1 - 
HHH000021: Bytecode provider name : javassist
2017-02-10 12:35:50 -0700 [INFO] from org.hibernate.annotations.common.Version in play-internal-execution-context-1 - 
HCANN000001: Hibernate Commons Annotations {4.0.5.Final}
2017-02-10 12:35:50 -0700 [INFO] from org.hibernate.dialect.Dialect in play-internal-execution-context-1 - 
HHH000400: Using dialect: org.hibernate.dialect.MySQLDialect
2017-02-10 12:35:50 -0700 [INFO] from org.hibernate.hql.internal.ast.ASTQueryTranslatorFactory in play-internal-execution-context-1 - 
HHH000397: Using ASTQueryTranslatorFactory
2017-02-10 12:35:51 -0700 [INFO] from org.hibernate.validator.internal.util.Version in play-internal-execution-context-1 - 
HV000001: Hibernate Validator 5.0.3.Final
2017-02-10 12:35:51 -0700 [INFO] from org.hibernate.tool.hbm2ddl.SchemaUpdate in play-internal-execution-context-1 - 
HHH000228: Running hbm2ddl schema update
2017-02-10 12:35:51 -0700 [INFO] from org.hibernate.tool.hbm2ddl.SchemaUpdate in play-internal-execution-context-1 - 
HHH000102: Fetching database metadata
2017-02-10 12:35:51 -0700 [INFO] from org.hibernate.tool.hbm2ddl.SchemaUpdate in play-internal-execution-context-1 - 
HHH000396: Updating schema
2017-02-10 12:35:51 -0700 [INFO] from org.hibernate.tool.hbm2ddl.TableMetadata in play-internal-execution-context-1 - 
HHH000261: Table found: advWeb.User
2017-02-10 12:35:51 -0700 [INFO] from org.hibernate.tool.hbm2ddl.TableMetadata in play-internal-execution-context-1 - 
HHH000037: Columns: [password, id, username]
2017-02-10 12:35:51 -0700 [INFO] from org.hibernate.tool.hbm2ddl.TableMetadata in play-internal-execution-context-1 - 
HHH000108: Foreign keys: []
2017-02-10 12:35:51 -0700 [INFO] from org.hibernate.tool.hbm2ddl.TableMetadata in play-internal-execution-context-1 - 
HHH000126: Indexes: [primary]
2017-02-10 12:35:51 -0700 [INFO] from org.hibernate.tool.hbm2ddl.SchemaUpdate in play-internal-execution-context-1 - 
HHH000232: Schema update complete
2017-02-10 12:35:56 -0700 [INFO] from play in play-internal-execution-context-1 - 
Application started (Dev)
2017-02-10 12:35:57 -0700 [INFO] from akka.event.slf4j.Slf4jLogger in play-akka.actor.default-dispatcher-3 - 
Slf4jLogger started
2017-02-10 12:35:57 -0700 [INFO] from controllers.Application in play-akka.actor.default-dispatcher-3 - 
Rendered Login page
2017-02-10 12:35:57 -0700 [INFO] from controllers.Application in play-akka.actor.default-dispatcher-4 - 
Rendered Login page
2017-02-10 12:36:03 -0700 [INFO] from controllers.Application in play-akka.actor.default-dispatcher-3 - 
Rendered Login page
2017-02-10 12:36:08 -0700 [INFO] from controllers.Application in play-akka.actor.default-dispatcher-3 - 
Rendered Login page
2017-02-10 12:37:23 -0700 [ERROR] from play in play-akka.actor.default-dispatcher-5 - 
Cannot invoke the action, eventually got an error: java.lang.RuntimeException: java.lang.reflect.InvocationTargetException
2017-02-10 12:37:23 -0700 [ERROR] from application in play-akka.actor.default-dispatcher-5 - 


! @732l715dn - Internal server error, for (POST) [/] ->

play.api.Application$$anon$1: Execution exception[[RuntimeException: java.lang.reflect.InvocationTargetException]]
	at play.api.Application$class.handleError(Application.scala:296) ~[play_2.11-2.3.10.jar:2.3.10]
	at play.api.DefaultApplication.handleError(Application.scala:402) [play_2.11-2.3.10.jar:2.3.10]
	at play.core.server.netty.PlayDefaultUpstreamHandler$$anonfun$3$$anonfun$applyOrElse$4.apply(PlayDefaultUpstreamHandler.scala:320) [play_2.11-2.3.10.jar:2.3.10]
	at play.core.server.netty.PlayDefaultUpstreamHandler$$anonfun$3$$anonfun$applyOrElse$4.apply(PlayDefaultUpstreamHandler.scala:320) [play_2.11-2.3.10.jar:2.3.10]
	at scala.Option.map(Option.scala:146) [scala-library.jar:na]
	at play.core.server.netty.PlayDefaultUpstreamHandler$$anonfun$3.applyOrElse(PlayDefaultUpstreamHandler.scala:320) [play_2.11-2.3.10.jar:2.3.10]
	at play.core.server.netty.PlayDefaultUpstreamHandler$$anonfun$3.applyOrElse(PlayDefaultUpstreamHandler.scala:316) [play_2.11-2.3.10.jar:2.3.10]
	at scala.concurrent.Future$$anonfun$recoverWith$1.apply(Future.scala:344) [scala-library.jar:na]
	at scala.concurrent.Future$$anonfun$recoverWith$1.apply(Future.scala:343) [scala-library.jar:na]
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:32) [scala-library.jar:na]
	at play.api.libs.iteratee.Execution$trampoline$.execute(Execution.scala:46) [play-iteratees_2.11-2.3.10.jar:2.3.10]
	at scala.concurrent.impl.CallbackRunnable.executeWithValue(Promise.scala:40) [scala-library.jar:na]
	at scala.concurrent.impl.Promise$DefaultPromise.tryComplete(Promise.scala:248) [scala-library.jar:na]
	at scala.concurrent.Promise$class.complete(Promise.scala:55) [scala-library.jar:na]
	at scala.concurrent.impl.Promise$DefaultPromise.complete(Promise.scala:153) [scala-library.jar:na]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.run(Future.scala:23) [scala-library.jar:na]
	at akka.dispatch.TaskInvocation.run(AbstractDispatcher.scala:41) [akka-actor_2.11-2.3.4.jar:na]
	at akka.dispatch.ForkJoinExecutorConfigurator$AkkaForkJoinTask.exec(AbstractDispatcher.scala:393) [akka-actor_2.11-2.3.4.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260) [scala-library.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339) [scala-library.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979) [scala-library.jar:na]
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107) [scala-library.jar:na]
Caused by: java.lang.RuntimeException: java.lang.reflect.InvocationTargetException
	at play.data.Form.bind(Form.java:394) ~[play-java_2.11-2.3.10.jar:2.3.10]
	at play.data.Form.bindFromRequest(Form.java:221) ~[play-java_2.11-2.3.10.jar:2.3.10]
	at controllers.Application.authenticate(Application.java:34) ~[classes/:na]
	at Routes$$anonfun$routes$1$$anonfun$applyOrElse$3$$anonfun$apply$3.apply(routes_routing.scala:90) ~[classes/:na]
	at Routes$$anonfun$routes$1$$anonfun$applyOrElse$3$$anonfun$apply$3.apply(routes_routing.scala:90) ~[classes/:na]
	at play.core.Router$HandlerInvokerFactory$$anon$4.resultCall(Router.scala:264) ~[play_2.11-2.3.10.jar:2.3.10]
	at play.core.Router$HandlerInvokerFactory$JavaActionInvokerFactory$$anon$15$$anon$1.invocation(Router.scala:255) ~[play_2.11-2.3.10.jar:2.3.10]
	at play.core.j.JavaAction$$anon$1.call(JavaAction.scala:55) ~[play_2.11-2.3.10.jar:2.3.10]
	at play.core.j.JavaAction$$anonfun$11.apply(JavaAction.scala:82) ~[play_2.11-2.3.10.jar:2.3.10]
	at play.core.j.JavaAction$$anonfun$11.apply(JavaAction.scala:82) ~[play_2.11-2.3.10.jar:2.3.10]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.liftedTree1$1(Future.scala:24) [scala-library.jar:na]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.run(Future.scala:24) [scala-library.jar:na]
	at play.core.j.HttpExecutionContext$$anon$2.run(HttpExecutionContext.scala:40) ~[play_2.11-2.3.10.jar:2.3.10]
	at play.api.libs.iteratee.Execution$trampoline$.execute(Execution.scala:46) [play-iteratees_2.11-2.3.10.jar:2.3.10]
	at play.core.j.HttpExecutionContext.execute(HttpExecutionContext.scala:32) ~[play_2.11-2.3.10.jar:2.3.10]
	at scala.concurrent.impl.Future$.apply(Future.scala:31) ~[scala-library.jar:na]
	at scala.concurrent.Future$.apply(Future.scala:492) ~[scala-library.jar:na]
	at play.core.j.JavaAction$class.apply(JavaAction.scala:82) ~[play_2.11-2.3.10.jar:2.3.10]
	at play.core.Router$HandlerInvokerFactory$JavaActionInvokerFactory$$anon$15$$anon$1.apply(Router.scala:252) ~[play_2.11-2.3.10.jar:2.3.10]
	at play.api.mvc.Action$$anonfun$apply$1$$anonfun$apply$4$$anonfun$apply$5.apply(Action.scala:130) ~[play_2.11-2.3.10.jar:2.3.10]
	at play.api.mvc.Action$$anonfun$apply$1$$anonfun$apply$4$$anonfun$apply$5.apply(Action.scala:130) ~[play_2.11-2.3.10.jar:2.3.10]
	at play.utils.Threads$.withContextClassLoader(Threads.scala:21) ~[play_2.11-2.3.10.jar:2.3.10]
	at play.api.mvc.Action$$anonfun$apply$1$$anonfun$apply$4.apply(Action.scala:129) ~[play_2.11-2.3.10.jar:2.3.10]
	at play.api.mvc.Action$$anonfun$apply$1$$anonfun$apply$4.apply(Action.scala:128) ~[play_2.11-2.3.10.jar:2.3.10]
	at scala.Option.map(Option.scala:146) [scala-library.jar:na]
	at play.api.mvc.Action$$anonfun$apply$1.apply(Action.scala:128) ~[play_2.11-2.3.10.jar:2.3.10]
	at play.api.mvc.Action$$anonfun$apply$1.apply(Action.scala:121) ~[play_2.11-2.3.10.jar:2.3.10]
	at play.api.libs.iteratee.Iteratee$$anonfun$mapM$1.apply(Iteratee.scala:483) ~[play-iteratees_2.11-2.3.10.jar:2.3.10]
	at play.api.libs.iteratee.Iteratee$$anonfun$mapM$1.apply(Iteratee.scala:483) ~[play-iteratees_2.11-2.3.10.jar:2.3.10]
	at play.api.libs.iteratee.Iteratee$$anonfun$flatMapM$1.apply(Iteratee.scala:519) ~[play-iteratees_2.11-2.3.10.jar:2.3.10]
	at play.api.libs.iteratee.Iteratee$$anonfun$flatMapM$1.apply(Iteratee.scala:519) ~[play-iteratees_2.11-2.3.10.jar:2.3.10]
	at play.api.libs.iteratee.Iteratee$$anonfun$flatMap$1$$anonfun$apply$14.apply(Iteratee.scala:496) ~[play-iteratees_2.11-2.3.10.jar:2.3.10]
	at play.api.libs.iteratee.Iteratee$$anonfun$flatMap$1$$anonfun$apply$14.apply(Iteratee.scala:496) ~[play-iteratees_2.11-2.3.10.jar:2.3.10]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.liftedTree1$1(Future.scala:24) [scala-library.jar:na]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.run(Future.scala:24) [scala-library.jar:na]
	... 6 common frames omitted
Caused by: java.lang.reflect.InvocationTargetException: null
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[na:1.8.0_121]
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[na:1.8.0_121]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[na:1.8.0_121]
	at java.lang.reflect.Method.invoke(Method.java:498) ~[na:1.8.0_121]
	at play.data.Form.bind(Form.java:391) ~[play-java_2.11-2.3.10.jar:2.3.10]
	... 40 common frames omitted
Caused by: java.lang.RuntimeException: No EntityManager bound to this thread. Try to annotate your action method with @play.db.jpa.Transactional
	at play.db.jpa.JPA.em(JPA.java:51) ~[play-java-jpa_2.11-2.3.10.jar:2.3.10]
	at assets.User.authenticate(User.java:63) ~[classes/:na]
	at controllers.Application$Login.validate(Application.java:58) ~[classes/:na]
	... 45 common frames omitted
2017-02-10 12:39:26 -0700 [INFO] from com.jolbox.bonecp.BoneCP in play-internal-execution-context-1 - 
Shutting down connection pool...
2017-02-10 12:39:26 -0700 [INFO] from com.jolbox.bonecp.BoneCP in play-internal-execution-context-1 - 
Connection pool has been shutdown.
2017-02-10 12:39:31 -0700 [INFO] from play in play-internal-execution-context-1 - 
datasource [jdbc:mysql://127.0.0.1/advWeb] bound to JNDI as DefaultDS
2017-02-10 12:39:31 -0700 [INFO] from play in play-internal-execution-context-1 - 
database [default] connected at jdbc:mysql://127.0.0.1/advWeb
2017-02-10 12:39:31 -0700 [WARN] from org.hibernate.ejb.HibernatePersistence in play-internal-execution-context-1 - 
HHH015016: Encountered a deprecated javax.persistence.spi.PersistenceProvider [org.hibernate.ejb.HibernatePersistence]; use [org.hibernate.jpa.HibernatePersistenceProvider] instead.
2017-02-10 12:39:31 -0700 [WARN] from org.hibernate.ejb.HibernatePersistence in play-internal-execution-context-1 - 
HHH015016: Encountered a deprecated javax.persistence.spi.PersistenceProvider [org.hibernate.ejb.HibernatePersistence]; use [org.hibernate.jpa.HibernatePersistenceProvider] instead.
2017-02-10 12:39:31 -0700 [WARN] from org.hibernate.ejb.HibernatePersistence in play-internal-execution-context-1 - 
HHH015016: Encountered a deprecated javax.persistence.spi.PersistenceProvider [org.hibernate.ejb.HibernatePersistence]; use [org.hibernate.jpa.HibernatePersistenceProvider] instead.
2017-02-10 12:39:31 -0700 [INFO] from org.hibernate.jpa.internal.util.LogHelper in play-internal-execution-context-1 - 
HHH000204: Processing PersistenceUnitInfo [
	name: defaultPersistenceUnit
	...]
2017-02-10 12:39:32 -0700 [INFO] from org.hibernate.dialect.Dialect in play-internal-execution-context-1 - 
HHH000400: Using dialect: org.hibernate.dialect.MySQLDialect
2017-02-10 12:39:32 -0700 [INFO] from org.hibernate.hql.internal.ast.ASTQueryTranslatorFactory in play-internal-execution-context-1 - 
HHH000397: Using ASTQueryTranslatorFactory
2017-02-10 12:39:32 -0700 [INFO] from org.hibernate.tool.hbm2ddl.SchemaUpdate in play-internal-execution-context-1 - 
HHH000228: Running hbm2ddl schema update
2017-02-10 12:39:32 -0700 [INFO] from org.hibernate.tool.hbm2ddl.SchemaUpdate in play-internal-execution-context-1 - 
HHH000102: Fetching database metadata
2017-02-10 12:39:32 -0700 [INFO] from org.hibernate.tool.hbm2ddl.SchemaUpdate in play-internal-execution-context-1 - 
HHH000396: Updating schema
2017-02-10 12:39:32 -0700 [INFO] from org.hibernate.tool.hbm2ddl.TableMetadata in play-internal-execution-context-1 - 
HHH000261: Table found: advWeb.User
2017-02-10 12:39:32 -0700 [INFO] from org.hibernate.tool.hbm2ddl.TableMetadata in play-internal-execution-context-1 - 
HHH000037: Columns: [password, id, username]
2017-02-10 12:39:32 -0700 [INFO] from org.hibernate.tool.hbm2ddl.TableMetadata in play-internal-execution-context-1 - 
HHH000108: Foreign keys: []
2017-02-10 12:39:32 -0700 [INFO] from org.hibernate.tool.hbm2ddl.TableMetadata in play-internal-execution-context-1 - 
HHH000126: Indexes: [primary]
2017-02-10 12:39:32 -0700 [INFO] from org.hibernate.tool.hbm2ddl.SchemaUpdate in play-internal-execution-context-1 - 
HHH000232: Schema update complete
2017-02-10 12:39:32 -0700 [INFO] from play in play-internal-execution-context-1 - 
Application started (Dev)
2017-02-10 12:39:32 -0700 [ERROR] from play in play-akka.actor.default-dispatcher-6 - 
Cannot invoke the action, eventually got an error: java.lang.RuntimeException: java.lang.reflect.InvocationTargetException
2017-02-10 12:39:32 -0700 [ERROR] from application in play-akka.actor.default-dispatcher-6 - 


! @732l78e6k - Internal server error, for (POST) [/] ->

play.api.Application$$anon$1: Execution exception[[RuntimeException: java.lang.reflect.InvocationTargetException]]
	at play.api.Application$class.handleError(Application.scala:296) ~[play_2.11-2.3.10.jar:2.3.10]
	at play.api.DefaultApplication.handleError(Application.scala:402) [play_2.11-2.3.10.jar:2.3.10]
	at play.core.server.netty.PlayDefaultUpstreamHandler$$anonfun$3$$anonfun$applyOrElse$4.apply(PlayDefaultUpstreamHandler.scala:320) [play_2.11-2.3.10.jar:2.3.10]
	at play.core.server.netty.PlayDefaultUpstreamHandler$$anonfun$3$$anonfun$applyOrElse$4.apply(PlayDefaultUpstreamHandler.scala:320) [play_2.11-2.3.10.jar:2.3.10]
	at scala.Option.map(Option.scala:146) [scala-library.jar:na]
	at play.core.server.netty.PlayDefaultUpstreamHandler$$anonfun$3.applyOrElse(PlayDefaultUpstreamHandler.scala:320) [play_2.11-2.3.10.jar:2.3.10]
	at play.core.server.netty.PlayDefaultUpstreamHandler$$anonfun$3.applyOrElse(PlayDefaultUpstreamHandler.scala:316) [play_2.11-2.3.10.jar:2.3.10]
	at scala.concurrent.Future$$anonfun$recoverWith$1.apply(Future.scala:344) [scala-library.jar:na]
	at scala.concurrent.Future$$anonfun$recoverWith$1.apply(Future.scala:343) [scala-library.jar:na]
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:32) [scala-library.jar:na]
	at play.api.libs.iteratee.Execution$trampoline$.execute(Execution.scala:46) [play-iteratees_2.11-2.3.10.jar:2.3.10]
	at scala.concurrent.impl.CallbackRunnable.executeWithValue(Promise.scala:40) [scala-library.jar:na]
	at scala.concurrent.impl.Promise$DefaultPromise.tryComplete(Promise.scala:248) [scala-library.jar:na]
	at scala.concurrent.Promise$class.complete(Promise.scala:55) [scala-library.jar:na]
	at scala.concurrent.impl.Promise$DefaultPromise.complete(Promise.scala:153) [scala-library.jar:na]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.run(Future.scala:23) [scala-library.jar:na]
	at akka.dispatch.TaskInvocation.run(AbstractDispatcher.scala:41) [akka-actor_2.11-2.3.4.jar:na]
	at akka.dispatch.ForkJoinExecutorConfigurator$AkkaForkJoinTask.exec(AbstractDispatcher.scala:393) [akka-actor_2.11-2.3.4.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260) [scala-library.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339) [scala-library.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979) [scala-library.jar:na]
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107) [scala-library.jar:na]
Caused by: java.lang.RuntimeException: java.lang.reflect.InvocationTargetException
	at play.data.Form.bind(Form.java:394) ~[play-java_2.11-2.3.10.jar:2.3.10]
	at play.data.Form.bindFromRequest(Form.java:221) ~[play-java_2.11-2.3.10.jar:2.3.10]
	at controllers.Application.authenticate(Application.java:34) ~[classes/:na]
	at Routes$$anonfun$routes$1$$anonfun$applyOrElse$3$$anonfun$apply$3.apply(routes_routing.scala:90) ~[classes/:na]
	at Routes$$anonfun$routes$1$$anonfun$applyOrElse$3$$anonfun$apply$3.apply(routes_routing.scala:90) ~[classes/:na]
	at play.core.Router$HandlerInvokerFactory$$anon$4.resultCall(Router.scala:264) ~[play_2.11-2.3.10.jar:2.3.10]
	at play.core.Router$HandlerInvokerFactory$JavaActionInvokerFactory$$anon$15$$anon$1.invocation(Router.scala:255) ~[play_2.11-2.3.10.jar:2.3.10]
	at play.core.j.JavaAction$$anon$1.call(JavaAction.scala:55) ~[play_2.11-2.3.10.jar:2.3.10]
	at play.core.j.JavaAction$$anonfun$11.apply(JavaAction.scala:82) ~[play_2.11-2.3.10.jar:2.3.10]
	at play.core.j.JavaAction$$anonfun$11.apply(JavaAction.scala:82) ~[play_2.11-2.3.10.jar:2.3.10]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.liftedTree1$1(Future.scala:24) [scala-library.jar:na]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.run(Future.scala:24) [scala-library.jar:na]
	at play.core.j.HttpExecutionContext$$anon$2.run(HttpExecutionContext.scala:40) ~[play_2.11-2.3.10.jar:2.3.10]
	at play.api.libs.iteratee.Execution$trampoline$.execute(Execution.scala:46) [play-iteratees_2.11-2.3.10.jar:2.3.10]
	at play.core.j.HttpExecutionContext.execute(HttpExecutionContext.scala:32) ~[play_2.11-2.3.10.jar:2.3.10]
	at scala.concurrent.impl.Future$.apply(Future.scala:31) ~[scala-library.jar:na]
	at scala.concurrent.Future$.apply(Future.scala:492) ~[scala-library.jar:na]
	at play.core.j.JavaAction$class.apply(JavaAction.scala:82) ~[play_2.11-2.3.10.jar:2.3.10]
	at play.core.Router$HandlerInvokerFactory$JavaActionInvokerFactory$$anon$15$$anon$1.apply(Router.scala:252) ~[play_2.11-2.3.10.jar:2.3.10]
	at play.api.mvc.Action$$anonfun$apply$1$$anonfun$apply$4$$anonfun$apply$5.apply(Action.scala:130) ~[play_2.11-2.3.10.jar:2.3.10]
	at play.api.mvc.Action$$anonfun$apply$1$$anonfun$apply$4$$anonfun$apply$5.apply(Action.scala:130) ~[play_2.11-2.3.10.jar:2.3.10]
	at play.utils.Threads$.withContextClassLoader(Threads.scala:21) ~[play_2.11-2.3.10.jar:2.3.10]
	at play.api.mvc.Action$$anonfun$apply$1$$anonfun$apply$4.apply(Action.scala:129) ~[play_2.11-2.3.10.jar:2.3.10]
	at play.api.mvc.Action$$anonfun$apply$1$$anonfun$apply$4.apply(Action.scala:128) ~[play_2.11-2.3.10.jar:2.3.10]
	at scala.Option.map(Option.scala:146) [scala-library.jar:na]
	at play.api.mvc.Action$$anonfun$apply$1.apply(Action.scala:128) ~[play_2.11-2.3.10.jar:2.3.10]
	at play.api.mvc.Action$$anonfun$apply$1.apply(Action.scala:121) ~[play_2.11-2.3.10.jar:2.3.10]
	at play.api.libs.iteratee.Iteratee$$anonfun$mapM$1.apply(Iteratee.scala:483) ~[play-iteratees_2.11-2.3.10.jar:2.3.10]
	at play.api.libs.iteratee.Iteratee$$anonfun$mapM$1.apply(Iteratee.scala:483) ~[play-iteratees_2.11-2.3.10.jar:2.3.10]
	at play.api.libs.iteratee.Iteratee$$anonfun$flatMapM$1.apply(Iteratee.scala:519) ~[play-iteratees_2.11-2.3.10.jar:2.3.10]
	at play.api.libs.iteratee.Iteratee$$anonfun$flatMapM$1.apply(Iteratee.scala:519) ~[play-iteratees_2.11-2.3.10.jar:2.3.10]
	at play.api.libs.iteratee.Iteratee$$anonfun$flatMap$1$$anonfun$apply$14.apply(Iteratee.scala:496) ~[play-iteratees_2.11-2.3.10.jar:2.3.10]
	at play.api.libs.iteratee.Iteratee$$anonfun$flatMap$1$$anonfun$apply$14.apply(Iteratee.scala:496) ~[play-iteratees_2.11-2.3.10.jar:2.3.10]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.liftedTree1$1(Future.scala:24) [scala-library.jar:na]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.run(Future.scala:24) [scala-library.jar:na]
	... 6 common frames omitted
Caused by: java.lang.reflect.InvocationTargetException: null
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[na:1.8.0_121]
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[na:1.8.0_121]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[na:1.8.0_121]
	at java.lang.reflect.Method.invoke(Method.java:498) ~[na:1.8.0_121]
	at play.data.Form.bind(Form.java:391) ~[play-java_2.11-2.3.10.jar:2.3.10]
	... 40 common frames omitted
Caused by: java.lang.RuntimeException: No EntityManager bound to this thread. Try to annotate your action method with @play.db.jpa.Transactional
	at play.db.jpa.JPA.em(JPA.java:51) ~[play-java-jpa_2.11-2.3.10.jar:2.3.10]
	at assets.User.authenticate(User.java:63) ~[classes/:na]
	at controllers.Application$Login.validate(Application.java:74) ~[classes/:na]
	... 45 common frames omitted
2017-02-10 12:39:36 -0700 [INFO] from controllers.Application in play-akka.actor.default-dispatcher-7 - 
Rendered Login page
2017-02-10 12:39:42 -0700 [ERROR] from play in play-akka.actor.default-dispatcher-8 - 
Cannot invoke the action, eventually got an error: java.lang.RuntimeException: java.lang.reflect.InvocationTargetException
2017-02-10 12:39:42 -0700 [ERROR] from application in play-akka.actor.default-dispatcher-8 - 


! @732l793g8 - Internal server error, for (POST) [/] ->

play.api.Application$$anon$1: Execution exception[[RuntimeException: java.lang.reflect.InvocationTargetException]]
	at play.api.Application$class.handleError(Application.scala:296) ~[play_2.11-2.3.10.jar:2.3.10]
	at play.api.DefaultApplication.handleError(Application.scala:402) [play_2.11-2.3.10.jar:2.3.10]
	at play.core.server.netty.PlayDefaultUpstreamHandler$$anonfun$3$$anonfun$applyOrElse$4.apply(PlayDefaultUpstreamHandler.scala:320) [play_2.11-2.3.10.jar:2.3.10]
	at play.core.server.netty.PlayDefaultUpstreamHandler$$anonfun$3$$anonfun$applyOrElse$4.apply(PlayDefaultUpstreamHandler.scala:320) [play_2.11-2.3.10.jar:2.3.10]
	at scala.Option.map(Option.scala:146) [scala-library.jar:na]
	at play.core.server.netty.PlayDefaultUpstreamHandler$$anonfun$3.applyOrElse(PlayDefaultUpstreamHandler.scala:320) [play_2.11-2.3.10.jar:2.3.10]
	at play.core.server.netty.PlayDefaultUpstreamHandler$$anonfun$3.applyOrElse(PlayDefaultUpstreamHandler.scala:316) [play_2.11-2.3.10.jar:2.3.10]
	at scala.concurrent.Future$$anonfun$recoverWith$1.apply(Future.scala:344) [scala-library.jar:na]
	at scala.concurrent.Future$$anonfun$recoverWith$1.apply(Future.scala:343) [scala-library.jar:na]
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:32) [scala-library.jar:na]
	at play.api.libs.iteratee.Execution$trampoline$.execute(Execution.scala:46) [play-iteratees_2.11-2.3.10.jar:2.3.10]
	at scala.concurrent.impl.CallbackRunnable.executeWithValue(Promise.scala:40) [scala-library.jar:na]
	at scala.concurrent.impl.Promise$DefaultPromise.tryComplete(Promise.scala:248) [scala-library.jar:na]
	at scala.concurrent.Promise$class.complete(Promise.scala:55) [scala-library.jar:na]
	at scala.concurrent.impl.Promise$DefaultPromise.complete(Promise.scala:153) [scala-library.jar:na]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.run(Future.scala:23) [scala-library.jar:na]
	at akka.dispatch.TaskInvocation.run(AbstractDispatcher.scala:41) [akka-actor_2.11-2.3.4.jar:na]
	at akka.dispatch.ForkJoinExecutorConfigurator$AkkaForkJoinTask.exec(AbstractDispatcher.scala:393) [akka-actor_2.11-2.3.4.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260) [scala-library.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339) [scala-library.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979) [scala-library.jar:na]
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107) [scala-library.jar:na]
Caused by: java.lang.RuntimeException: java.lang.reflect.InvocationTargetException
	at play.data.Form.bind(Form.java:394) ~[play-java_2.11-2.3.10.jar:2.3.10]
	at play.data.Form.bindFromRequest(Form.java:221) ~[play-java_2.11-2.3.10.jar:2.3.10]
	at controllers.Application.authenticate(Application.java:34) ~[classes/:na]
	at Routes$$anonfun$routes$1$$anonfun$applyOrElse$3$$anonfun$apply$3.apply(routes_routing.scala:90) ~[classes/:na]
	at Routes$$anonfun$routes$1$$anonfun$applyOrElse$3$$anonfun$apply$3.apply(routes_routing.scala:90) ~[classes/:na]
	at play.core.Router$HandlerInvokerFactory$$anon$4.resultCall(Router.scala:264) ~[play_2.11-2.3.10.jar:2.3.10]
	at play.core.Router$HandlerInvokerFactory$JavaActionInvokerFactory$$anon$15$$anon$1.invocation(Router.scala:255) ~[play_2.11-2.3.10.jar:2.3.10]
	at play.core.j.JavaAction$$anon$1.call(JavaAction.scala:55) ~[play_2.11-2.3.10.jar:2.3.10]
	at play.core.j.JavaAction$$anonfun$11.apply(JavaAction.scala:82) ~[play_2.11-2.3.10.jar:2.3.10]
	at play.core.j.JavaAction$$anonfun$11.apply(JavaAction.scala:82) ~[play_2.11-2.3.10.jar:2.3.10]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.liftedTree1$1(Future.scala:24) [scala-library.jar:na]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.run(Future.scala:24) [scala-library.jar:na]
	at play.core.j.HttpExecutionContext$$anon$2.run(HttpExecutionContext.scala:40) ~[play_2.11-2.3.10.jar:2.3.10]
	at play.api.libs.iteratee.Execution$trampoline$.execute(Execution.scala:46) [play-iteratees_2.11-2.3.10.jar:2.3.10]
	at play.core.j.HttpExecutionContext.execute(HttpExecutionContext.scala:32) ~[play_2.11-2.3.10.jar:2.3.10]
	at scala.concurrent.impl.Future$.apply(Future.scala:31) ~[scala-library.jar:na]
	at scala.concurrent.Future$.apply(Future.scala:492) ~[scala-library.jar:na]
	at play.core.j.JavaAction$class.apply(JavaAction.scala:82) ~[play_2.11-2.3.10.jar:2.3.10]
	at play.core.Router$HandlerInvokerFactory$JavaActionInvokerFactory$$anon$15$$anon$1.apply(Router.scala:252) ~[play_2.11-2.3.10.jar:2.3.10]
	at play.api.mvc.Action$$anonfun$apply$1$$anonfun$apply$4$$anonfun$apply$5.apply(Action.scala:130) ~[play_2.11-2.3.10.jar:2.3.10]
	at play.api.mvc.Action$$anonfun$apply$1$$anonfun$apply$4$$anonfun$apply$5.apply(Action.scala:130) ~[play_2.11-2.3.10.jar:2.3.10]
	at play.utils.Threads$.withContextClassLoader(Threads.scala:21) ~[play_2.11-2.3.10.jar:2.3.10]
	at play.api.mvc.Action$$anonfun$apply$1$$anonfun$apply$4.apply(Action.scala:129) ~[play_2.11-2.3.10.jar:2.3.10]
	at play.api.mvc.Action$$anonfun$apply$1$$anonfun$apply$4.apply(Action.scala:128) ~[play_2.11-2.3.10.jar:2.3.10]
	at scala.Option.map(Option.scala:146) [scala-library.jar:na]
	at play.api.mvc.Action$$anonfun$apply$1.apply(Action.scala:128) ~[play_2.11-2.3.10.jar:2.3.10]
	at play.api.mvc.Action$$anonfun$apply$1.apply(Action.scala:121) ~[play_2.11-2.3.10.jar:2.3.10]
	at play.api.libs.iteratee.Iteratee$$anonfun$mapM$1.apply(Iteratee.scala:483) ~[play-iteratees_2.11-2.3.10.jar:2.3.10]
	at play.api.libs.iteratee.Iteratee$$anonfun$mapM$1.apply(Iteratee.scala:483) ~[play-iteratees_2.11-2.3.10.jar:2.3.10]
	at play.api.libs.iteratee.Iteratee$$anonfun$flatMapM$1.apply(Iteratee.scala:519) ~[play-iteratees_2.11-2.3.10.jar:2.3.10]
	at play.api.libs.iteratee.Iteratee$$anonfun$flatMapM$1.apply(Iteratee.scala:519) ~[play-iteratees_2.11-2.3.10.jar:2.3.10]
	at play.api.libs.iteratee.Iteratee$$anonfun$flatMap$1$$anonfun$apply$14.apply(Iteratee.scala:496) ~[play-iteratees_2.11-2.3.10.jar:2.3.10]
	at play.api.libs.iteratee.Iteratee$$anonfun$flatMap$1$$anonfun$apply$14.apply(Iteratee.scala:496) ~[play-iteratees_2.11-2.3.10.jar:2.3.10]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.liftedTree1$1(Future.scala:24) [scala-library.jar:na]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.run(Future.scala:24) [scala-library.jar:na]
	... 6 common frames omitted
Caused by: java.lang.reflect.InvocationTargetException: null
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[na:1.8.0_121]
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[na:1.8.0_121]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[na:1.8.0_121]
	at java.lang.reflect.Method.invoke(Method.java:498) ~[na:1.8.0_121]
	at play.data.Form.bind(Form.java:391) ~[play-java_2.11-2.3.10.jar:2.3.10]
	... 40 common frames omitted
Caused by: java.lang.RuntimeException: No EntityManager bound to this thread. Try to annotate your action method with @play.db.jpa.Transactional
	at play.db.jpa.JPA.em(JPA.java:51) ~[play-java-jpa_2.11-2.3.10.jar:2.3.10]
	at assets.User.authenticate(User.java:63) ~[classes/:na]
	at controllers.Application$Login.validate(Application.java:74) ~[classes/:na]
	... 45 common frames omitted
2017-02-10 12:40:46 -0700 [INFO] from com.jolbox.bonecp.BoneCP in play-internal-execution-context-1 - 
Shutting down connection pool...
2017-02-10 12:40:46 -0700 [INFO] from com.jolbox.bonecp.BoneCP in play-internal-execution-context-1 - 
Connection pool has been shutdown.
2017-02-10 12:40:51 -0700 [INFO] from play in play-internal-execution-context-1 - 
datasource [jdbc:mysql://127.0.0.1/advWeb] bound to JNDI as DefaultDS
2017-02-10 12:40:51 -0700 [INFO] from play in play-internal-execution-context-1 - 
database [default] connected at jdbc:mysql://127.0.0.1/advWeb
2017-02-10 12:40:51 -0700 [WARN] from org.hibernate.ejb.HibernatePersistence in play-internal-execution-context-1 - 
HHH015016: Encountered a deprecated javax.persistence.spi.PersistenceProvider [org.hibernate.ejb.HibernatePersistence]; use [org.hibernate.jpa.HibernatePersistenceProvider] instead.
2017-02-10 12:40:51 -0700 [WARN] from org.hibernate.ejb.HibernatePersistence in play-internal-execution-context-1 - 
HHH015016: Encountered a deprecated javax.persistence.spi.PersistenceProvider [org.hibernate.ejb.HibernatePersistence]; use [org.hibernate.jpa.HibernatePersistenceProvider] instead.
2017-02-10 12:40:51 -0700 [WARN] from org.hibernate.ejb.HibernatePersistence in play-internal-execution-context-1 - 
HHH015016: Encountered a deprecated javax.persistence.spi.PersistenceProvider [org.hibernate.ejb.HibernatePersistence]; use [org.hibernate.jpa.HibernatePersistenceProvider] instead.
2017-02-10 12:40:51 -0700 [INFO] from org.hibernate.jpa.internal.util.LogHelper in play-internal-execution-context-1 - 
HHH000204: Processing PersistenceUnitInfo [
	name: defaultPersistenceUnit
	...]
2017-02-10 12:40:51 -0700 [INFO] from org.hibernate.dialect.Dialect in play-internal-execution-context-1 - 
HHH000400: Using dialect: org.hibernate.dialect.MySQLDialect
2017-02-10 12:40:51 -0700 [INFO] from org.hibernate.hql.internal.ast.ASTQueryTranslatorFactory in play-internal-execution-context-1 - 
HHH000397: Using ASTQueryTranslatorFactory
2017-02-10 12:40:51 -0700 [INFO] from org.hibernate.tool.hbm2ddl.SchemaUpdate in play-internal-execution-context-1 - 
HHH000228: Running hbm2ddl schema update
2017-02-10 12:40:51 -0700 [INFO] from org.hibernate.tool.hbm2ddl.SchemaUpdate in play-internal-execution-context-1 - 
HHH000102: Fetching database metadata
2017-02-10 12:40:51 -0700 [INFO] from org.hibernate.tool.hbm2ddl.SchemaUpdate in play-internal-execution-context-1 - 
HHH000396: Updating schema
2017-02-10 12:40:51 -0700 [INFO] from org.hibernate.tool.hbm2ddl.TableMetadata in play-internal-execution-context-1 - 
HHH000261: Table found: advWeb.User
2017-02-10 12:40:51 -0700 [INFO] from org.hibernate.tool.hbm2ddl.TableMetadata in play-internal-execution-context-1 - 
HHH000037: Columns: [password, id, username]
2017-02-10 12:40:51 -0700 [INFO] from org.hibernate.tool.hbm2ddl.TableMetadata in play-internal-execution-context-1 - 
HHH000108: Foreign keys: []
2017-02-10 12:40:51 -0700 [INFO] from org.hibernate.tool.hbm2ddl.TableMetadata in play-internal-execution-context-1 - 
HHH000126: Indexes: [primary]
2017-02-10 12:40:51 -0700 [INFO] from org.hibernate.tool.hbm2ddl.SchemaUpdate in play-internal-execution-context-1 - 
HHH000232: Schema update complete
2017-02-10 12:40:51 -0700 [INFO] from play in play-internal-execution-context-1 - 
Application started (Dev)
2017-02-10 12:40:51 -0700 [INFO] from controllers.Application in play-akka.actor.default-dispatcher-9 - 
Rendered Login page
2017-02-10 12:40:58 -0700 [INFO] from controllers.Application in play-akka.actor.default-dispatcher-13 - 
New session created for subin
2017-02-10 12:40:58 -0700 [INFO] from controllers.Application in play-akka.actor.default-dispatcher-17 - 
Rendered Home page
2017-02-10 12:41:55 -0700 [INFO] from com.jolbox.bonecp.BoneCP in pool-5-thread-1 - 
Shutting down connection pool...
2017-02-10 12:41:55 -0700 [INFO] from com.jolbox.bonecp.BoneCP in pool-5-thread-1 - 
Connection pool has been shutdown.
